### ISSUE TITLE
Merge `Id` and `Event` messages

### ISSUE THREAD

**Issue by goto-bus-stop** [goto-bus-stop/sse-codec#2]
Per discussion in #1

Event can just have an `Option<String>` or an empty `String` for its `data` member if the `id` is the only thing that's sent.

**Comment by bekh6ex** [comment]
Sorry, got back to it rather late :)

Just to emphasize the need, this is what I need to do now:
https://github.com/bekh6ex/wikidata-realtime-dumps/blob/master/src/events/event_stream.rs#L29-L41
https://github.com/bekh6ex/wikidata-realtime-dumps/blob/master/src/events/mod.rs#L154-L167

**Comment by goto-bus-stop** [comment]
Released in :package: 0.3.0!

`sse-codec` now also remembers the `id` between different messages, so if you get something like:

```
id: id1
data: somemessage

data: nextmessage
```
it will emit:

```rust
Event::Message {
    event: "message",
    data: "somemessage",
    id: Some("id1"),
},
Event::Message {
    event: "message",
    data: "nextmessage",
    id: Some("id1"),
}
```
